service/model/project_membership.go

```diff
@@ -11,28 +11,41 @@ import (
 )
 
 const (
+	ProjectRoleOwner  ProjectRole = "owner"
```

Comment:
I added the project role `owner`, even though it is not currently needed for granting privileges since all project admin privileges are based on the user role admin. 

However, it is still useful to identify who created the project. We might extend privileges in the future, and having this role will be useful.